{"tag": "function", "depth": 1, "time": 1610354218.9281883, "data": {"name": "shell", "call_args": {"cmd": "am force-stop com.kugou.shiqutouch"}, "start_time": 1610354218.072768, "ret": "", "end_time": 1610354218.9281883}}
{"tag": "function", "depth": 1, "time": 1610354220.163702, "data": {"name": "start_app", "call_args": {"package": "com.kugou.shiqutouch", "activity": null}, "start_time": 1610354218.9281883, "ret": null, "end_time": 1610354220.163702}}
{"tag": "function", "depth": 1, "time": 1610354225.163757, "data": {"name": "sleep", "call_args": {"secs": 5}, "start_time": 1610354220.163702, "ret": null, "end_time": 1610354225.163757}}
{"tag": "function", "depth": 1, "time": 1610354225.5044339, "data": {"name": "record_ui", "call_args": {"driver": "<poco.drivers.android.uiautomation.AndroidUiautomationPoco object at 0x00000222000A9FC8>", "action": "'click'", "ui": "UIObjectProxy of \"com.kugou.shiqutouch:id/activity_url_hunter_tips_close\"", "args": "[0.9, 0.0625]"}, "start_time": 1610354225.5044339, "ret": "UIObjectProxy of \"com.kugou.shiqutouch:id/activity_url_hunter_tips_close\"", "end_time": 1610354225.5044339}}
{"tag": "function", "depth": 2, "time": 1610354233.1185024, "data": {"name": "try_log_screen", "call_args": {"screen": null}, "start_time": 1610354228.8259656, "ret": {"screen": "1610354232990.jpg", "resolution": [1080, 2160]}, "end_time": 1610354233.1185024}}
{"tag": "function", "depth": 1, "time": 1610354237.2922482, "data": {"name": "touch", "call_args": {"v": [972.0, 135.0], "kwargs": {"duration": 0.01}, "times": 1}, "start_time": 1610354228.8259656, "ret": [972.0, 135.0], "end_time": 1610354237.2922482}}
{"tag": "info", "depth": 0, "time": 1610354245.3410058, "data": {"name": "Final Error", "traceback": "Traceback (most recent call last):\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\poco\\proxy.py\", line 26, in wrapped\n    proxy.wait_for_appearance(timeout=proxy.poco._pre_action_wait_for_appearance)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\poco\\proxy.py\", line 681, in wait_for_appearance\n    raise PocoTargetTimeout('appearance', self)\npoco.exceptions.PocoTargetTimeout: Waiting timeout for appearance of \"UIObjectProxy of \"com.kugou.shiqutouch:id/dialog_problem_guid_close\"\"\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\airtest\\cli\\runner.py\", line 61, in runTest\n    exec(compile(code.encode(\"utf-8\"), pyfilepath, 'exec'), self.scope)\n  File \"D:\\CC\\program\\cx_test_tool\\Folder\\APP_xn_auto_folder\\air\\enterApp.air\\enterApp.py\", line 24, in <module>\n    poco(\"com.kugou.shiqutouch:id/dialog_problem_guid_close\").swipe([0.0039, 0.0])\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\poco\\proxy.py\", line 29, in wrapped\n    raise e\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\poco\\proxy.py\", line 23, in wrapped\n    return func(proxy, *args, **kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\poco\\proxy.py\", line 457, in swipe\n    origin = self.get_position(focus)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\poco\\proxy.py\", line 72, in wrapped\n    return func(proxy, *args, **kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\poco\\proxy.py\", line 613, in get_position\n    pos = self.attr('pos')\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\poco\\proxy.py\", line 39, in wrapped\n    return func(self, *args, **kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\poco\\proxy.py\", line 734, in attr\n    nodes = self._do_query(multiple=False)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\poco\\proxy.py\", line 874, in _do_query\n    raise PocoNoSuchNodeException(self)\npoco.exceptions.PocoNoSuchNodeException: Cannot find any visible node by query UIObjectProxy of \"com.kugou.shiqutouch:id/dialog_problem_guid_close\"\n"}}
